check_sums.tsv

```diff
@@ -3,14 +3,14 @@
 d4486dde14da14b4f8887a7415e2866f  data/BRCARNASeqClin.tsv
 1a89ea769381e300e5a88ec61713ad9e  data/BRCAarray.pcl
 02e72c33071307ff6570621480d3c90b  data/EBPlusPlusAdjustPANCAN_IlluminaHiSeq_RNASeqV2.geneExp.tsv
-90feb5edb48d7619568d2de21697509e  data/GBMClin.tsv
+ce911443e071c4251fb3f196780e76de  data/GBMClin.tsv
```

Comment:
This, and the change below, is related to 

> "G-CIMP" into "GCIMP"

Correct?

Reply:
Yes that's right